homeassistant/components/gree/switch.py

```diff
@@ -13,6 +16,45 @@
 from .entity import GreeEntity
 
 
+@dataclass
+class GreeSwitchRequiredKeysMixin:
+    """Mixin for require keys."""
+
+    property: Props
```

Comment:
If you use the "key" with `getattr`/`setattr`, then you can drop this `Mixin` class.

Reply:
Done... and the tests pass so hopefully it's all good! :)